### PR TITLE
[Push] Lint as framework automatically.

### PR DESCRIPTION
**Pull request by neonichu**
See CocoaPods/CocoaPods#3168

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## Master
+
+##### Enhancements
+
+* Lint as framework automatically. If needed, `--use-libraries` option
```
Comment (segiddins) [inline CHANGELOG.md]: `as a`

Comment (segiddins) [inline CHANGELOG.md]: the `--`

Comment (neonichu) [inline CHANGELOG.md]: :uk:

**Comment by segiddins** [comment]
:+1: once travis runs

**Comment by orta** [comment]
if travis runs

**Comment by neonichu** [comment]
Run, travis, run!

**Pull request merged**